kuma/core/helpers.py

```diff
@@ -150,8 +151,11 @@ def entity_decode(str):
 
 
 @register.function
-def inlinei18n(locale):
-    return StaticI18nJob().get(locale)
+def statici18n(locale):
+    if not locale:
```

Comment:
It uses the `static` template tag behind the scene (https://github.com/zyegfryed/django-statici18n/blob/87f70fa732b7a671660703e7096fead0f7a592dd/src/statici18n/templatetags/statici18n.py#L6): https://github.com/django/django/blob/9a2aca60304dc2e98f9ef45636e129d225cb981f/django/contrib/staticfiles/templatetags/staticfiles.py